docs/reference/client-side-encryption.txt

```diff
@@ -389,9 +390,16 @@ use that information to generate a data key.
 
 If you have created an Azure master key, note the tenant id, the client id, and
 the client secret of the application that has permissions to use the key.
-Additionally, note the key name, key version (id any), and key vault endpoint
+Additionally, note the key name, key version (if any), and key vault endpoint
 for your master key. You will use that information to generate a data key.
 
+If you have created an GCP KMS master key, note the email and the private key
+the client secret of the application that has permissions to use the key.
```

Comment:
I think grammar is broken between the previous line and this line.

Reply:
Fixed, thanks.